python/lsst/faro/measurement/DetectorTableMeasurement.py

```diff
@@ -0,0 +1,129 @@
+import os
+
+import lsst.pipe.base as pipeBase
+from lsst.verify.tasks import MetricConnections
+import lsst.pex.config as pexConfig
+import lsst.geom
+from lsst.utils import getPackageDir
+
+from lsst.faro.base.CatalogMeasurementBase import CatalogMeasurementBaseTaskConfig, CatalogMeasurementBaseTask
+from lsst.pipe.tasks.loadReferenceCatalog import LoadReferenceCatalogConfig, LoadReferenceCatalogTask
+
+__all__ = ("DetectorTableMeasurementTaskConfig", "DetectorTableMeasurementTask")
+
+
+class DetectorTableMeasurementTaskConnections(MetricConnections,
+                                              dimensions=("instrument", "visit", "detector", "band"),
+                                              defaultTemplates={"refDataset": ""}):
+
+    catalog = pipeBase.connectionTypes.Input(doc="Source catalog for visit.",
+                                             dimensions=("instrument", "visit", "band"),
+                                             storageClass="DataFrame",
+                                             name="sourceTable_visit",
+                                             deferLoad=True)
+
+    refcat = pipeBase.connectionTypes.PrerequisiteInput(
+        doc="Reference catalog.",
```

Comment:
I think I'd prefer the indentation for this attribute match the others in this file for readability

Reply:
Making consistent.